src/Microsoft.AspNetCore.Razor.Design/build/netstandard2.0/Microsoft.AspNetCore.Razor.Design.targets

```diff
@@ -52,7 +52,7 @@
 
   <Target
     Name="_RazorResolveTagHelpers"
-    DependsOnTargets="CoreCompile;RazorResolveAssemblyReferences"
+    DependsOnTargets="Compile;RazorResolveAssemblyReferences"
```

Comment:
So `CoreCompile` doesn't run `ResolveReferences` which causes the compilation to fail. Is running `/t:RazorCodeGen` by itself a scenario we would support?

Reply:
I don't understand this comment. We need `ResolveReferences` to run to get taghelpers.

Reply:
Here's what the target call stack looks like:
```
RazorCodeGen 
  _RazorResolveTagHelpers
    CoreCompile
      Csc -> Errors since references haven't been resolved
      
RazorCodeGen 
  _RazorResolveTagHelpers
    Compile
        ResolveReferences
        CoreCompile
          Csc -> Works
```

Reply:
Cool! Yeah this was just a bug.